sdm_versions_manager/database.py

```diff
@@ -0,0 +1,80 @@
+# Contains functions to connect to MongoDB, create the database, and perform CRUD operations.
+
+from pymongo import MongoClient
+import os
```

Comment:
use only the required functions change import os by from os import getenv for example